api/providers/ClientsProvider.cs

```diff
@@ -43,6 +43,18 @@ public ClientsProvider(AppDbContext db, IValidator<Client> clientValidator, Cont
 
         return newClient;
     }
+    public override Client? Delete(Guid id)
+    {
+        Client? foundClient = GetById(id);
+
+        if (foundClient == null) return null;
+
```

Comment:
checken als een client gekoppeld is aan iets voor dat je verwijderd bijvoorbeeld shipments